packages/voting/src/voting.rs

```diff
@@ -56,11 +56,15 @@ impl MultipleChoiceVotes {
     }
 
     pub fn add_vote(&mut self, vote: MultipleChoiceVote, weight: Uint128) {
-        self.vote_weights[vote.option_id as usize] += weight;
+        self.vote_weights[vote.option_id as usize] = self.vote_weights[vote.option_id as usize]
+            .checked_add(weight)
+            .unwrap();
```

Comment:
Instead of doing unwrap here, let's map it to overflow error similar to here: https://github.com/DA0-DA0/dao-contracts/blob/c37d7d76713e27f2ba86b41b31810b8279022237/contracts/cw20-stake/src/contract.rs#L258

Sorry, should have been more clear in original comment. The reason we want to do this is so that we avoid panics and return useful errors